functions/venue.js

```diff
@@ -113,7 +112,6 @@ const createVenueData = (data, context) => {
     case VenueTemplate.partymap:
     case VenueTemplate.themecamp:
       venueData.rooms = data.rooms;
-      venueData.mapBackgroundImageUrl = data.mapBackgroundImageUrl;
```

Comment:
This seems to be the source of the bug raised in https://github.com/sparkletown/internal-sparkle-issues/issues/131 with fix in https://github.com/sparkletown/sparkle/pull/1071